### ISSUE TITLE
Thumbnails not showing after update 

### ISSUE THREAD

**Issue by akhilleusuggo** [WWBN/AVideo#2575]
![image](https://user-images.githubusercontent.com/52116577/73455938-a5346680-4379-11ea-9c46-14ccbd8baecf.png)

I confirm that the files does exist on /videos

**Comment by DanielnetoDotCom** [comment]
What thumbs?

please screenshot

**Comment by akhilleusuggo** [comment]
![image](https://user-images.githubusercontent.com/52116577/73498683-a8a20f00-43c6-11ea-94cf-b54b92301850.png)

**Comment by DanielnetoDotCom** [comment]
Clear your cache, in case it persists inspect the code and check what URL you have in your images.

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom 
![image](https://user-images.githubusercontent.com/52116577/73541076-3539e500-443a-11ea-987b-d4004f3d532c.png)
You can take a look by yourself

**Comment by DanielnetoDotCom** [comment]
Is that a jpg?

I can not see the extension.

can you directly open this image in your browser?

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom No I can't .
This is the site , you can inspect it by yourself . https://tube.akhilleusuggo.com/
This happened only after update , and I don't know why

**Comment by DanielnetoDotCom** [comment]
Ok

that is your image  https://tube.akhilleusuggo.com/videos/_YPTuniqid_5e1401e9761ca0.04870522_thumbsSmallV2.jpg

it is a 0 bytes file, so something is happening you can not reach that.

you must check your access.log for some clue

also remove your .htaccess just to make sure it is not blocking the image. if you remove the .htaccess and still an error on directly access the image your problem is somewhere else, you must check your apache configs

**Comment by akhilleusuggo** [comment]
```
@DanielnetoDotCom 
/var/www/html/YouPHPTube/videos# du -h _YPTuniqid_5e1401e9761ca0.04870522_thumbsSmallV2.jpg
12K     _YPTuniqid_5e1401e9761ca0.04870522_thumbsSmallV2.jpg
```

**Comment by DanielnetoDotCom** [comment]
Ok, check the access.log and the .htaccess

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom 
```
"GET /videos/_YPTuniqid_5e1401e9761ca0.04870522_thumbsSmallV2.jpg HTTP/1.1" 200 666 "https://github.com/WWBN/AVideo/issues/2575" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36"
xx.xx.xx.xx - - [31/Jan/2020:14:10:35 +0000] "GET /plugin/Live/stats.json.php?Menu HTTP/1.1" 200 598 "https://tube.akhilleusuggo.com/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36"
```

**Comment by DanielnetoDotCom** [comment]
.htaccess now

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom Just to put things clear , cause I didn't understood . the .htaccess on the apache2 or the AVideo script ?

**Comment by DanielnetoDotCom** [comment]
AVideo directory, I am conserning more about line 243, you can try to comment it out.

![image](https://user-images.githubusercontent.com/18009293/73546416-f4cb6e80-441b-11ea-8619-ba6bf76a6a8e.png)

**Comment by akhilleusuggo** [comment]
```
1  # BEGIN Expires Don't forget to enable mod_headers and mod_expires, you can do so by running a2enmod rewrite and a2enmod headers
     2  <ifModule mod_expires.c>
     3      ExpiresActive On
     4      ExpiresDefault "access plus 1 seconds"
     5      ExpiresByType text/html "access plus 1 seconds"
     6      ExpiresByType image/gif "access plus 2592000 seconds"
     7      ExpiresByType image/jpeg "access plus 2592000 seconds"
     8      ExpiresByType image/png "access plus 2592000 seconds"
     9      ExpiresByType text/css "access plus 604800 seconds"
    10      ExpiresByType text/javascript "access plus 216000 seconds"
    11      ExpiresByType application/x-javascript "access plus 216000 seconds"
    12  </ifModule>
    13  <Files routes.php>
    14  Order Allow,Deny
    15  Deny from all
    16  </Files>
    17  <Files PayPal.log>
    18  Order Allow,Deny
    19  Deny from all
    20  </Files>
    21  <Files avideo.log>
    22  Order Allow,Deny
    23  Deny from all
    24  </Files>
    25  <Files avideo.js.log>
    26  Order Allow,Deny
    27  Deny from all
    28  </Files>
    29  # Disable directory browsing
    30  Options All -Indexes
    31  # END Expires
    32  # BEGIN Caching
    33  <ifModule mod_headers.c>
    34      <FilesMatch "\.(ts|key|ttf|otf|eot|woff|woff2)$">
    35          Header set Access-Control-Allow-Origin: *
    36      </FilesMatch>
    37      <filesMatch "\.(ico|pdf|flv|jpg|jpeg|png|gif|swf)$">
    38          Header set Cache-Control "max-age=2592000, public"
    39      </filesMatch>
    40      <filesMatch "\.(css)$">
    41          Header set Cache-Control "max-age=604800, public"
    42      </filesMatch>
    43      <filesMatch "\.(js)$">
    44          Header set Cache-Control "max-age=216000, private"
    45      </filesMatch>
    46      <filesMatch "\.(xml|txt)$">
    47          Header set Cache-Control "max-age=216000, public, must-revalidate"
    48      </filesMatch>
    49      <filesMatch "\.(html|htm|php)$">
    50          Header set Cache-Control "max-age=1, private, must-revalidate"
    51      </filesMatch>
    52  </ifModule>
    53  # END Caching
    54  <IfModule mod_rewrite.c>
    55      RewriteEngine on
    56
    57      RewriteRule    glyphicons-halflings-regular(.+)$ view/bootstrap/fonts/glyphicons-halflings-regular$1    [NC,L]
    58
    59      #main Files
    60      RewriteRule    ^index.php$ view/index.php    [NC,L]
    61      #RewriteRule    ^index.php$ /view/index.php    [NC,L]
    62      #RewriteRule    ^$ view/    [NC,L]
    63      RewriteRule    ^bootstrap/(.+)$ view/bootstrap/$1 [NC,L]
    64      RewriteRule    ^js/(.+)$ view/js/$1 [NC,L]
    65      RewriteRule    ^css/(.+)$ view/css/$1 [NC,L]
    66      RewriteRule    ^img/(.+)$ view/img/$1 [NC,L]
    67      RewriteRule    ^videos/fonts/(.+)$ view/bootstrap/fonts/$1 [NC,L]
    68      RewriteRule    ^page/([0-9]+)/?$ view/?page=$1    [QSA]
    69      RewriteRule    ^videoOnly/?$ view/?type=video    [NC,L]
    70      RewriteRule    ^audioOnly/?$ view/?type=audio    [NC,L]
    71      RewriteRule    ^download$ view/downloadExternalVideo.php    [NC,L]
    72      RewriteRule    ^info$ view/info.php    [NC,L]
    73      RewriteRule    ^downloadNow$ objects/downloadVideo.php    [NC,L]
    74      RewriteRule    ^getDownloadProgress$ objects/downloadVideoProgress.php    [NC,L]
    75      RewriteRule    ^status$ objects/status.json.php    [NC,L]
    76      RewriteRule    ^menu/([A-Za-z0-9-_.]+)/? plugin/TopMenu/seo.php?menuSeoUrlItem=$1    [QSA,L]
    77      RewriteRule    ^admin/plugin/(.+)$ plugin/$1 [NC,L]
    78      RewriteRule    ^trending/?$ view/trending.php    [NC,L,QSA]
    79
    80      RewriteRule    ^about$ view/about.php    [NC,L]
    81      RewriteRule    ^contact$ view/contact.php    [NC,L]
    82      RewriteRule    ^sendEmail$ objects/sendEmail.json.php    [NC,L]
    83      RewriteRule    ^captcha$ objects/getCaptcha.php    [NC,L]
    84      RewriteRule    ^monitor/(.+)$ objects/ServerMonitor/$1    [NC,L]
    85      RewriteRule    ^videosList$ view/videosList.php    [NC,L]
    86      RewriteRule    ^videosList/video/([A-Za-z0-9-_.]+)/page/([0-9]+)/?$ view/videosList.php?videoName=$1&page=$2    [QSA]
    87      RewriteRule    ^videosList/cat/([A-Za-z0-9-]+)/video/([A-Za-z0-9-_.]+)/page/([0-9]+)/?$ view/videosList.php?catName=$1&videoName=$2&page=$3    [QSA]
    88
    89      #for the category name
    90      RewriteRule    ^cat/([A-Za-z0-9-]+)/?$ view/?catName=$1    [NC,L,QSA]
    91      RewriteRule    ^cat/([A-Za-z0-9-]+)/page/([0-9]+)/?$ view/?catName=$1&page=$2    [NC,L,QSA]
    92
    93      RewriteRule    ^tag/([0-9-]+)(.*)?$ view/?tags_id=$1    [NC,L,QSA]
    94
    95
    96      #for the video name
    97      RewriteRule    ^video/([A-Za-z0-9-_.]+)/?$ view/?videoName=$1    [QSA]
    98      RewriteRule    ^video/([A-Za-z0-9-_.]+)/page/([0-9]+)/??$ view/?videoName=$1&page=$2    [QSA]
    99      RewriteRule    ^v/([0-9]+)/?$ view/?v=$1    [QSA]
   100      RewriteRule    ^v/([0-9]+)/page/([0-9]+)/??$ view/?v=$1&page=$2     [QSA]
   101      RewriteRule    ^video/([0-9]+)/([A-Za-z0-9-_.]+)/?$ view/?v=$1    [QSA]
   102      RewriteRule    ^video/([0-9]+)/([A-Za-z0-9-_.]+)/page/([0-9]+)/??$ view/?v=$1&page=$3     [QSA]
   103
   104      #for the video name
   105      RewriteRule    ^cat/([A-Za-z0-9-]+)/video/([A-Za-z0-9-_.]+)/?$ view/?catName=$1&videoName=$2    [QSA]
   106      RewriteRule    ^cat/([A-Za-z0-9-]+)/video/([A-Za-z0-9-_.]+)/page/([0-9]+)/?$ view/?catName=$1&videoName=$2&page=$3   [QSA]
   107      #for the embeded video name
   108      RewriteRule    ^videoEmbeded/([0-9]+)/? view/videoEmbeded.php?v=$1    [QSA]
   109      RewriteRule    ^videoEmbeded/([A-Za-z0-9-_.]+)/?$ view/videoEmbeded.php?videoName=$1    [QSA]
   110      RewriteRule    ^cat/([A-Za-z0-9-_.]+)/videoEmbeded/([A-Za-z0-9-_.]+)/?$ view/videoEmbeded.php?catName=$1&videoName=$2    [QSA]
   111      RewriteRule    ^videoEmbed/([0-9]+)/? view/videoEmbeded.php?v=$1    [QSA]
   112      RewriteRule    ^videoEmbed/([A-Za-z0-9-_.]+)/?$ view/videoEmbeded.php?videoName=$1    [QSA]
   113      RewriteRule    ^vEmbed/([0-9]+)/?$ view/videoEmbeded.php?v=$1    [QSA]
   114      RewriteRule    ^videoEmbed/([0-9]+)/([A-Za-z0-9-_.]+)/?$ view/videoEmbeded.php?v=$1    [QSA]
   115
   116      RewriteRule    ^plugin/([A-Za-z0-9-_.]+)/(.*)?$ plugin/$1/$2    [NC,L]
   117
   118      RewriteRule    ^upload$ view/mini-upload-form/    [NC,L]
   119      RewriteRule    ^fileUpload$ view/mini-upload-form/upload.php    [NC,L]
   120      RewriteRule    ^uploadPoster/([0-9]+)/(jpg|gif)$ objects/uploadPoster.php?video_id=$1&type=$2    [NC,L]
   121
   122      #edit your own user
   123      RewriteRule    ^user$ view/user.php    [NC,L]
   124
   125      #manager user
   126      RewriteRule    ^users$ view/managerUsers.php    [NC,L]
   127      RewriteRule    ^users.json$ objects/users.json.php    [NC,L]
   128      RewriteRule    ^updateUser$ objects/userUpdate.json.php    [NC,L]
   129      RewriteRule    ^savePhoto$ objects/userSavePhoto.php    [NC,L]
   130      RewriteRule    ^saveBackground$ objects/userSaveBackground.php    [NC,L]
   131      RewriteRule    ^addNewUser$ objects/userAddNew.json.php    [NC,L]
   132      RewriteRule    ^deleteUser$ objects/userDelete.json.php    [NC,L]
   133      RewriteRule    ^recoverPass$ objects/userRecoverPass.php    [NC,L]
   134      RewriteRule    ^saveRecoverPassword$ objects/userRecoverPassSave.json.php    [NC,L]
   135      RewriteRule    ^signUp$ view/signUp.php    [NC,L]
   136      RewriteRule    ^createUser$ objects/userCreate.json.php    [NC,L]
   137
   138
   139      RewriteRule    ^usersGroups$ view/managerUsersGroups.php    [NC,L]
   140      RewriteRule    ^usersGroups.json$ objects/usersGroups.json.php    [NC,L]
   141      RewriteRule    ^addNewUserGroups$ objects/userGroupsAddNew.json.php    [NC,L]
   142      RewriteRule    ^deleteUserGroups$ objects/userGroupsDelete.json.php    [NC,L]
   143
   144      #manager category
   145      RewriteRule    ^categories$ view/managerCategories.php    [NC,L]
   146      RewriteRule    ^categories.json$ objects/categories.json.php    [NC,L]
   147      RewriteRule    ^addNewCategory$ objects/categoryAddNew.json.php    [NC,L]
   148      RewriteRule    ^deleteCategory$ objects/categoryDelete.json.php    [NC,L]
   149
   150      #manager plugin
   151      RewriteRule    ^plugins$ view/managerPlugins.php    [NC,L]
   152      RewriteRule    ^plugins.json$ objects/plugins.json.php    [NC,L]
   153      RewriteRule    ^pluginsAvailable.json$ objects/pluginsAvailable.json.php    [NC,L]
   154      RewriteRule    ^pluginImport.json$ objects/pluginImport.json.php    [NC,L]
   155      RewriteRule    ^switchPlugin$ objects/pluginSwitch.json.php    [NC,L]
   156      RewriteRule    ^addDataObjectPlugin.json$ objects/pluginAddDataObject.json.php    [NC,L]
   157      RewriteRule    ^runDBScriptPlugin.json$ objects/pluginRunDatabaseScript.json.php    [NC,L]
   158
   159      #manager playList
   160      RewriteRule    ^playLists.json$ objects/playlists.json.php    [NC,L]
   161      RewriteRule    ^playListsVideos.json$ objects/playlistsVideos.json.php    [NC,L]
   162      RewriteRule    ^playListsFromUser.json/([0-9]+)/?$ objects/playlistsFromUser.json.php?users_id=$1    [NC,L]
   163      RewriteRule    ^addNewPlayList$ objects/playlistAddNew.json.php    [NC,L]
   164      RewriteRule    ^playListAddVideo.json$ objects/playListAddVideo.json.php    [NC,L]
   165      RewriteRule    ^playlist/([0-9]+)/([0-9]+)/?$ view/?playlist_id=$1&playlist_index=$2    [NC,L]
   166      RewriteRule    ^playlist/([0-9]+)/?$ view/?playlist_id=$1    [NC,L]
   167      RewriteRule    ^removeVideoFromPlaylist/?$ objects/playlistRemoveVideo.php    [NC,L]
   168      RewriteRule    ^removePlaylist/?$ objects/playlistRemove.php    [NC,L]
   169      RewriteRule    ^renamePlaylist/?$ objects/playlistRename.php    [NC,L]
   170      RewriteRule    ^sortPlaylist/?$ objects/playlistSort.php    [NC,L]
   171      RewriteRule    ^channel/([^/]+)/feed/?$ feed/index.php?channelName=$1    [QSA]
   172      RewriteRule    ^channel/([^/]+) view/channel.php?channelName=$1    [QSA]
   173      RewriteRule    ^channel/?$ view/channel.php    [QSA]
   174      RewriteRule    ^channels/?$ view/channels.php    [NC,L]
   175
   176      #playlist alias
   177      RewriteRule    ^programs.json$ objects/playlists.json.php    [QSA]
   178      RewriteRule    ^program/([0-9]+)/([0-9]+)(/.*)?$ view/?playlist_id=$1&playlist_index=$2    [QSA]
   179      RewriteRule    ^program/([0-9]+)(/.*)?$ view/?playlist_id=$1    [QSA]
   180      RewriteRule    ^viewProgram/([0-9]+).*?$ view/channelProgram.php?program_id=$1    [QSA]
   181
   182      #manager videos
   183      RewriteRule    ^orphanFiles$ view/orphanFiles.php    [NC,L]
   184      RewriteRule    ^mvideos$ view/managerVideos.php    [NC,L]
   185      RewriteRule    ^videos.json$ objects/videos.json.php    [NC,L]
   186      RewriteRule    ^videosAndroid.json$ objects/videosAndroid.json.php    [NC,L]
   187      RewriteRule    ^videoAndroid.json$ objects/videoAndroid.json.php    [NC,L]
   188      RewriteRule    ^deleteVideo$ objects/videoDelete.json.php    [NC,L]
   189      RewriteRule    ^addNewVideo$ objects/videoAddNew.json.php    [NC,L]
   190      RewriteRule    ^refreshVideo$ objects/videoRefresh.json.php    [NC,L]
   191      RewriteRule    ^setStatusVideo$ objects/videoStatus.json.php    [NC,L]
   192      RewriteRule    ^setCategoryVideo$ objects/videoCategory.json.php    [NC,L]
   193      RewriteRule    ^rotateVideo$ objects/videoRotate.json.php    [NC,L]
   194
   195
   196      # Subscribes
   197      RewriteRule    ^subscribes$ view/managerSubscribes.php    [NC,L]
   198      RewriteRule    ^subscribes.json$ objects/subscribes.json.php    [NC,L]
   199      RewriteRule    ^subscribe.json$ objects/subscribe.json.php    [NC,L]
   200      RewriteRule    ^notifySubscribers.json$ objects/notifySubscribers.json.php    [NC,L]
   201
   202
   203      RewriteRule    ^aVideoQueueEncoder.json$ objects/aVideoQueueEncoder.json.php    [NC,L]
   204      RewriteRule    ^aVideoEncoder.json$ objects/aVideoEncoder.json.php    [NC,L]
   205
   206      #comment
   207      RewriteRule    ^comments$ view/managerComments.php    [NC,L]
   208      RewriteRule    ^saveComment$ objects/commentAddNew.json.php    [NC,L]
   209      RewriteRule    ^comments.json/([0-9]+)$ objects/comments.json.php?video_id=$1    [NC,L]
   210
   211      RewriteRule    ^login$ objects/login.json.php    [NC,L]
   212      RewriteRule    ^logoff$ objects/logoff.php    [NC,L]
   213
   214      RewriteRule    ^like$ objects/like.json.php?like=1    [QSA]
   215      RewriteRule    ^dislike$ objects/like.json.php?like=-1    [QSA]
   216
   217
   218      #manager configuration
   219
   220      RewriteRule    ^update/?$ view/update.php    [NC,L]
   221      RewriteRule    ^siteConfigurations$ view/configurations.php    [NC,L]
   222      RewriteRule    ^updateConfig$ objects/configurationUpdate.json.php    [NC,L]
   223
   224      RewriteRule    ^charts$ view/charts.php    [NC,L]
   225
   226      RewriteRule    ^help$ view/help.php    [NC,L]
   227
   228      # YouTube Sync
   229      RewriteRule    ^youtubeUpload$ objects/youtubeUpload.json.php    [NC,L]
   230
   231      RewriteRule    ^googleAdView$ view/googleAdView.php    [NC,L]
   232
   233      RewriteRule    ^notifications.json$ objects/notifications.json.php    [NC,L]
   234
   235      RewriteRule    ^sitemap.xml$ view/sitemap.xml.php    [NC,L]
   236      RewriteRule    ^robots.txt$ view/robots.txt.php    [NC,L]
   237
   238      RewriteRule    ^videos/(.*)/index.m3u8$ view/hls.php?videoDirectory=$1    [NC,L,QSA]
   239
   240      RewriteRule    ^vast.xml/?([0-9]+)?$ plugin/AD_Server/VAST.php?campaign_has_videos_id=$1    [NC,L]
   241      # It must be before image not found
   242      <IfModule mod_xsendfile.c>
   243         RewriteRule    ^videos/([A-Za-z0-9-_.]+)$ view/xsendfile.php?file=$1    [QSA]
   244      </IfModule>
   245
   246      # if image do not exists
   247      RewriteCond %{REQUEST_URI} \.(jpg|jpeg|gif|png|ico)$ [NC]
   248      RewriteCond %{REQUEST_FILENAME} !-f
   249      RewriteCond %{REQUEST_FILENAME} !-d
   250      RewriteRule .*$ img/image404.php [NC,L]
   251
   252      RewriteCond %{REQUEST_FILENAME} !-f
   253      RewriteCond %{REQUEST_FILENAME} !-d
   254      RewriteCond %{REQUEST_FILENAME} !-l
   255      RewriteRule .*$ view/error.php [NC,L]
   256
   257  </IfModule>
   258  <IfModule mod_deflate.c>
   259    # Compress HTML, CSS, JavaScript, Text, XML and fonts
   260    AddOutputFilterByType DEFLATE application/javascript
   261    AddOutputFilterByType DEFLATE application/rss+xml
   262    AddOutputFilterByType DEFLATE application/vnd.ms-fontobject
   263    AddOutputFilterByType DEFLATE application/x-font
   264    AddOutputFilterByType DEFLATE application/x-font-opentype
   265    AddOutputFilterByType DEFLATE application/x-font-otf
   266    AddOutputFilterByType DEFLATE application/x-font-truetype
   267    AddOutputFilterByType DEFLATE application/x-font-ttf
   268    AddOutputFilterByType DEFLATE application/x-javascript
   269    AddOutputFilterByType DEFLATE application/xhtml+xml
   270    AddOutputFilterByType DEFLATE application/xml
   271    AddOutputFilterByType DEFLATE font/opentype
   272    AddOutputFilterByType DEFLATE font/otf
   273    AddOutputFilterByType DEFLATE font/ttf
   274    AddOutputFilterByType DEFLATE image/svg+xml
   275    AddOutputFilterByType DEFLATE image/x-icon
   276    AddOutputFilterByType DEFLATE text/css
   277    AddOutputFilterByType DEFLATE text/html
   278    AddOutputFilterByType DEFLATE text/javascript
   279    AddOutputFilterByType DEFLATE text/plain
   280    AddOutputFilterByType DEFLATE text/xml
   281

```
I want you to look into this : 
```
<IfModule mod_xsendfile.c>
       RewriteRule    ^videos/([A-Za-z0-9-_.]+)$ view/xsendfile.php?file=$1    [QSA]
    </IfModule>
```
It is well positioned ? I got the plugin for SecureVideosDirectory , didn't configured it yet ( and disabled )

**Comment by akhilleusuggo** [comment]
Oh , so we're on the same .
So I have to comment the 3 lines ?

**Comment by akhilleusuggo** [comment]
Fixed my commenting the 3 lines 
It's strange , cause the same .htacess using it for over 6 months and I remember this option was there .
Thank you for the help !

**Comment by DanielnetoDotCom** [comment]
I think you should first rename the .htaccess file and check if your image loads.

if does not load it is something outside AVideo, if loads comment the line 243 and test again

**Comment by DanielnetoDotCom** [comment]
ok, so your xsendfile is not properly configured

Run this `sudo apt-get install libapache2-mod-xsendfile && sudo a2enmod xsendfile`

Configure your apache XSendFile in your apache2.conf or in your virtual host config

```
<Directory /var/www/html/YouPHPTube/>
    Options Indexes FollowSymLinks
    XSendFile on
    XSendFilePath /var/www/html/YouPHPTube/
    AllowOverride All
    Require all granted
    Order Allow,Deny
    Allow from All
</Directory>
```

Make sure you restart your apache

**Comment by DanielnetoDotCom** [comment]
if you are not using the SecureVideosDirectory Plugin just comment it out and ignore the configuration

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom 
```
Reading package lists... Done
Building dependency tree
Reading state information... Done
libapache2-mod-xsendfile is already the newest version (0.12-2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Module xsendfile already enabled

```
I'll open a new issue when I'll want to enable it . 
For the moment I have the .htaccess like this 

```
#<IfModule mod_xsendfile.c>
         #RewriteRule    ^videos/([A-Za-z0-9-_.]+)$ view/xsendfile.php?file=$1    [QSA]
#</IfModule>
```

You're suggestion to uncomment 
`RewriteRule    ^videos/([A-Za-z0-9-_.]+)$ view/xsendfile.php?file=$1    [QSA]`

Right ?

**Comment by DanielnetoDotCom** [comment]
Uncomment only if you are using SecureVideosDirectory Plugin, otherwise it will be useless.

I just sent an update to not use xsendfile for images, only for videos and audio

**Comment by akhilleusuggo** [comment]
Thank you @DanielnetoDotCom
